### ISSUE TITLE
latest container fails to start with option ordering error

### ISSUE THREAD

**Issue by jaychris** [itzg/dockerfiles#92]
the lead node starts fine, but subsequent nodes, when passing in UNICAST_HOSTS, reports this error:

ERROR: Parameter [-Des.discovery.zen.ping.unicast.hosts=es0] starting with "-D", "-d" or "-p" must be before any parameters starting with --

Here is the startup log:

```
Starting Elasticsearch with the options -Dnetwork.host=0.0.0.0 -Des.path.conf=/conf   -Des.path.data=/data   -Des.path.logs=/data   -Des.transport.tcp.port=9300   -Des.http.port=9200   --path.plugins=/conf/plugins -Des.discovery.zen.ping.unicast.hosts=es0
Running as non-root...
+ su -c /usr/share/elasticsearch-2.3.4/bin/elasticsearch -Dnetwork.host=0.0.0.0 -Des.path.conf=/conf   -Des.path.data=/data   -Des.path.logs=/data   -Des.transport.tcp.port=9300   -Des.http.port=9200   --path.plugins=/conf/plugins -Des.discovery.zen.ping.unicast.hosts=es0 - elasticsearch
ERROR: Parameter [-Des.discovery.zen.ping.unicast.hosts=es0] starting with "-D", "-d" or "-p" must be before any parameters starting with --
```

and here is the command used to start the container:

`docker run -it --rm --name es1 -h es1 -p 9200:9200 -p 9300:9300 -e OPTS=-Dnetwork.host=0.0.0.0 -e UNICAST_HOSTS=es0 -e ES_HEAP_SIZE=8g -v /data:/data itzg/elasticsearch`

**Comment by itzg** [comment]
Oops, I knew about the ordering, but forgot to inspect what the rest of the script would do.

**Comment by jaychris** [comment]
I've been able to workaround for now by specifying the options directly as part of OPTS since that gets ordered first, like:

-e "OPTS= -Dnetwork.host=0.0.0.0 -Des.discovery.zen.ping.unicast.hosts=es0"

**Comment by itzg** [comment]
Glad you found a workaround since I'm going to be afk until tomorrow afternoon. It looks like the easy fix is similar where it should use the -D properties style at

    --path.plugins=/conf/plugins"

Eventually I want to swap those all out for the new style arguments.

**Comment by jaychris** [comment]
fixed at sha256:99f265f5a774e3120d1a311e1bb42983137a180b283ce0a90a6b379eb04caa8f

**Comment by itzg** [comment]
fixed by 87b2ce1c2466e209bc78876b5513aa270e24a852
